common/appointment/get.ts

```diff
@@ -0,0 +1,103 @@
+import { Lecture as Appointment } from '../../graphql/generated';
+import { prisma } from '../prisma';
+import { User } from '../user';
+
+export const getAppointmentsForUser = async (
+    user: User,
+    take?: number,
+    skip?: number,
+    cursor?: number,
+    direction?: 'last' | 'next'
+): Promise<Appointment[]> => {
+    let appointments;
+
+    if (!direction && !cursor) {
+        return await getAppointmentsByIdList(user.userID, take, skip);
+    }
+
+    if (!direction || !cursor) {
+        throw Error('Cursor or direction not specified for cursor based pagination');
+    }
+
+    if (direction === 'next' && cursor) {
+        appointments = await getNextAppointmentsByIdList(user.userID, take, cursor);
+    }
+
+    if (direction === 'last' && cursor) {
+        appointments = await getLastAppointmentsByIdList(user.userID, take, cursor);
+    }
+
+    return appointments;
```

Comment:
I think we can just return `getLastAppointmentsByIdList(user.userID, take, cursor);` here, because direction only has three states where two are covered already.